src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,110 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> entering;
```

Comment:
"Your implementation should have two pointers to the first and the last Node."

Reply:
what is `entering` variable? Do we need it? Lets use just `first` and `last` nodes